core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -1352,8 +1370,7 @@ class ReplicaManagerTest {
       Optional.of(1))
     val fetchResult = sendConsumerFetch(replicaManager, tp0, partitionData, None, timeout = 10)
     assertNull(fetchResult.get)
-
-    Mockito.when(replicaManager.metadataCache.contains(tp0)).thenReturn(true)
+    Mockito.when(replicaManager.metadataCache.contains(ArgumentMatchers.eq(tp0))).thenReturn(true)
```

Comment:
```suggestion
    Mockito.when(metadataCache.contains(ArgumentMatchers.eq(tp0))).thenReturn(true)
```

Reply:
your proposed change doesn't compile because metadataCache is not defined in this scope